### ISSUE TITLE
CTA does not have sufficient color contrast

### ISSUE THREAD

**Issue by KaterinaKir** [wmcadigital/wmn-design-system#42]
Element has insufficient color contrast of 4.34 (foreground color: #1d7ec4, background color: #ffffff, font size: 12.0pt (16px), font weight: bold). Expected contrast ratio of 4.5:1

**Comment by KaterinaKir** [comment]
New CTA colour is #1D7BBF
Passed contrast checker
